OpenBudgeteer.Core/ViewModels/PageViewModels/ReportPageViewModel.cs

```diff
@@ -68,11 +68,12 @@ protected async Task<List<Tuple<DateOnly, decimal>>> LoadMonthBalancesAsync(int
         {
             var currentMonth = new DateOnly(DateTime.Today.Year, DateTime.Today.Month, 1);
 
-            var transactions = ServiceManager.BankTransactionService
+            var transactions = ServiceManager.BudgetedTransactionService
                 .GetAll(currentMonth.AddMonths((months - 1) * -1), DateOnly.MaxValue)
```

Comment:
Just as an idea, maybe it makes sense to create a dedicated endpoint in the Service layer like  `GetAllForReporting(DateOnly periodStart, DateOnly periodEnd)`

Reply:
Good Idea. FYI: I intentionally didn't use this function in the `Bank Balances` graph in order to show the actual bank balance there.

Reply:
I already noticed that and thought about exactly the same thing :smile: